packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesService.kt

```diff
@@ -42,6 +42,8 @@ open class UpdatesService(protected var context: Context) : InternalModule, Upda
     get() = UpdatesController.instance.databaseHolder
   override val isEmergencyLaunch: Boolean
     get() = UpdatesController.instance.isEmergencyLaunch
+  override val isEmbeddedLaunch: Boolean
+    get() = launchedUpdate != null && embeddedUpdate != null && launchedUpdate?.id?.equals(embeddedUpdate?.id) ?: false
```

Comment:
```suggestion
    get() = launchedUpdate != null && embeddedUpdate != null && launchedUpdate.id?.equals(embeddedUpdat?.id) ?: false
```
If the first two statements already guard the `null` conditions, I don't think the optional chaining is necessary in the last condition.

Reply:
Android Studio insisted on it anyway 😄 That said, I'll check to see if there is a better Kotlin expression for this

Reply:
Found that it can be much simpler